# Test version used by test_mock_gps_node.launch.py
# Intentionally comments out and changes some values
mock_gps_node:
  ros__parameters:
    wms:
      url: 'http://localhost:80/wms'
      #version: '1.1.1'
      layers: ['imagery']
      styles: ['']
      srs: 'EPSG:4326'
      image_format: 'image/jpeg'
      request_timeout: 10
    misc:
      autopilot: 'gisnav.autopilots.px4_micrortps.PX4microRTPS'
      image_topic: '/camera/image_raw'
      camera_info_topic: '/camera/camera_info'
      mock_gps_selection: 1
      attitude_deviation_threshold: 10
      #max_pitch: 30
      #min_match_altitude: 80
      min_match_altitude: 70  # Non-default value
    map_update:
      update_delay: 1
      gimbal_projection: True
      max_pitch: 30
      max_map_radius: 1000
      update_map_area_threshold: 0.85
    #pose_estimator:
    #  params_file: 'config/loftr_params.yaml'
    #debug:
    #  export_position: 'position.json'
    #  export_projection: 'projection.json'